examples/classification/README.md

```diff
@@ -34,3 +38,89 @@ python classification.py mobilenet_v2 --batch_size 8 --num_cores 4
 ```
 
 Run with the `-h` flag to see all available models.
+
+## Benchmarking Example
+`benchmark.py` is a script for benchmarking sparsified image classification model performance with DeepSparse. For a full list of options run `python benchmark.py -h`.
+To run a benchmark using the DeepSparse Engine with a moderately pruned resnet model that uses all available CPU cores and batch size 1, run:
+
+```bash
+python benchmark.py "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
+```
+
+Example using own model and data:
+
+```bash
+python benchmark.py path_to_onnx_file \
+--data-path path_to_data_directory
+```
+
+Replace `path_to_onnx_file` with [SparseZoo](http://sparsezoo.neuralmagic.com/) stub or filepath to ONNX model file and replace `path_to_data_directory` with appropriate SparseZoo stub or path to a directory containing `.npz` files to be used as model inputs.
+## Example Image Classification DeepSparse Flask Deployment
+To illustrate how the DeepSparse Engine can be used with image classification models.
+The server uses Flask to create an app with the DeepSparse Engine hosting a compiled image classification model. The client can make requests into the server returning inference results for given inputs.
+
+### Server
+First, start up the host server.py with your model of choice, SparseZoo stubs are also supported.
+
+Example command:
+```bash
+python server.py "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
+```
+
+You can leave that running as a detached process or in a spare terminal.
+
+This starts a Flask app with the DeepSparse Engine as the inference backend, accessible at http://0.0.0.0:5543 by default.
+
+The app exposes HTTP endpoints at:
```

Comment:
^ this line and the next line don't make sense; do you mean that the HTTP endpoints now allow you to run these two commands?

Reply:
This means once the server is running, our app creates two urls 

```
http://0.0.0.0:5543/info : to get information about which models are being hosted
http://0.0.0.0:5543/predict : to post inputs and get inference results
``` 

Here http://0.0.0.0:5543 is the default url, user can also specify their own url while running server.py

Reply:
^ I think your explanation here is great and what I would clarify for completeness.

Once the server is running, our app creates two URLs, exposing the following HTTP endpoints:
<list two out>
Feel free to specify your own URL while running `server.py`.

Reply:
👌  done!